test/SparqlAntlrParserTestHelpers.h

```diff
@@ -242,3 +242,5 @@ MATCHER_P(GroupByVariablesMatch, vars, "") {
   return std::equal(groupVariables.begin(), groupVariables.end(), vars.begin(),
                     [](auto& var, auto& var1) { return var.name() == var1; });
 }
+
+MATCHER_P(IsPropertyPath, propertyPath, "") { return arg == propertyPath; }
```

Comment:
Is it correct, that this trivial matcher is needed, because the `expectCompleteParse` expects a matcher? If so, then comment this here, otherwise people might wonder why this is needed.

Reply:
Yes.